crawler.py

```diff
@@ -212,6 +212,7 @@ def get_git_info(path):
                 "  TLDs to exclude: %s\n"
                 "  Firefox ETP: %s\n"
                 "  driver capabilities:\n\n%s\n"
+                "  parallel extensions: %s\n"
```

Comment:
Let's log `--load-extension` higher up in the list, so that driver capabilities, a large object, is the final entry.

Reply:
Why "extensions"? One or none, right?